.travis.yml

```diff
@@ -15,12 +15,7 @@ script:
   - yarn css:build
   - yarn test -- --coverage
   - yarn build
-  - docker login -u $DOCKER_USER -p $DOCKER_PASS
-  - export REPO=masinio/scrumlr
-  - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH ; fi`
-  - docker build -f Dockerfile -t $REPO:$TRAVIS_COMMIT .
-  - docker tag $REPO:$TRAVIS_COMMIT $REPO:$TAG
-  - docker push $REPO
+  - ./scripts/travis-ci-push-docker-image.sh
```

Comment:
How is this handled correctly? And why is this implementation different from the one in #5 and #6?
Can you please isolate this change from the others and open a separate PR for it? That would be awesome. 👍